logger.js

```diff
@@ -0,0 +1,205 @@
+const levels = {
+  info: "INFO",
+  warn: "WARN",
+  error: "ERROR",
+  debug: "DEBUG"
+}
+
+var currentLogLevel = "info"
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider removing unused variable declarations</b></div>
<div id="fix">

It appears that `currentLogLevel` is declared but never referenced in the code. Similarly, functions `clearLogs` and `dummyLoggerFunction` are defined but not called anywhere. Consider removing these unused declarations to improve code maintainability, or document their intended purpose if they're planned for future use.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -8,1 +8,0 @@
 -var currentLogLevel = "info"

```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/aakashgusain1990/BitoPythonTest/pull/21#issuecomment-2440543133>#378142</a></i></small>
</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged